java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -267,4 +272,53 @@ public void info_with_empty_InfoOptions_returns_success() {
         assertEquals(testResponse, response);
         assertEquals(testPayload, payload);
     }
+
+    @SneakyThrows
+    @Test
+    public void mget_returns_success() {
+        // setup
+        String[] keys = {"key1", null, "key2"};
+        String[] values = {"value1", null, "value2"};
+
+        CompletableFuture testResponse = mock(CompletableFuture.class);
+        when(testResponse.thenApply(any())).thenReturn(testResponse);
+        when(testResponse.get()).thenReturn(values);
+
+        // match on protobuf request
+        when(commandManager.<String>submitNewCommand(eq(MGet), eq(keys), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<String[]> response = service.mget(keys);
+        String[] payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(values, payload);
+    }
+
+    @SneakyThrows
+    @Test
+    public void mset_returns_success() {
+        // setup
+        Map<String, String> keyValueMap = new LinkedHashMap<>();
+        keyValueMap.put("key1", "value1");
+        keyValueMap.put("key2", "value2");
+        String[] args = {"key1", "value1", "key2", "value2"};
+
+        CompletableFuture testResponse = mock(CompletableFuture.class);
```

Comment:
what about `when(testResponse.thenApply(any())).thenReturn(testResponse);`

Reply:
Fixed not required anymore.